auth/src/api/builder.rs

```diff
@@ -62,24 +62,24 @@ impl Default for ApiBuilder {
 impl ApiBuilder {
     pub fn new() -> Self {
         let router = Router::new()
-            .route("/", get(health_check))
+            .route("/", get(|| async move {})) // Health check: 200 OK
             .route("/auth/key", get(convert_key))
-            .route("/auth/refresh", post(refresh_token))
+            .route("/auth/refresh", post(|| async move {}))
```

Comment:
Does anything even call this?

Reply:
I believe this is a remnant of the skeleton we created when we started working on the auth service, we never ended up using it. I cannot see any calls to this endpoint in our Datadog history either.